test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -100,162 +97,161 @@ describe('<AddressSection>', () => {
   it('should show addressUpdateUnavailable if countries or states lists aren\'t available', () => {
     const props = Object.assign({}, defaultProps, { countriesAvailable: false });
-    const component = ReactTestUtils.renderIntoDocument(<AddressSection {...props}/>);
-    const tree = getFormDOM(component);
+    const tree = mount(<AddressSection {...props}/>);
 
     // Start editing
-    tree.click('button.usa-button-outline');
-    expect(tree.getElement('.usa-alert-heading').textContent).to.contain('Address update unavailable');
+    tree.find('button.usa-button-outline').simulate('click');
+    expect(tree.find('.usa-alert-heading').text()).to.contain('Address update unavailable');
   });
 
   it('should load address in new props after mounting', () => {
     const props = cloneDeep(defaultProps);
     props.savedAddress = {};
-    const tree = SkinDeep.shallowRender(<AddressSection {...props}/>);
+    const tree = mount(<AddressSection {...props}/>);
 
-    const instance = tree.getMountedInstance();
-    expect(instance.state.editableAddress).to.equal(props.savedAddress);
+    expect(tree.state('editableAddress')).to.equal(props.savedAddress);
+    tree.find('.usa-button-outline').simulate('click');
 
-    const newProps = Object.assign({}, props, { savedAddress: { addressOne: '123 Main St' } });
-    instance.componentWillReceiveProps(newProps);
-    expect(instance.state.editableAddress).to.equal(newProps.savedAddress);
+    // Edit the street address
+    const newAddress = '123 Main St';
+    tree.find('input[name="addressOne"]').simulate('change', { target: { value: newAddress } });
+    expect(tree.state('editableAddress').addressOne).to.equal(newAddress);
   });
 
   it('should not call saveAddress when Cancel is clicked', () => {
     saveSpy.reset();
-    const component = ReactTestUtils.renderIntoDocument(<AddressSection {...defaultProps}/>);
-    const tree = getFormDOM(component);
+    const tree = mount(<AddressSection {...defaultProps}/>);
 
     // Start editing
-    tree.click('button.usa-button-outline');
+    tree.find('button.usa-button-outline').simulate('click');
 
     // Try to save
-    tree.click('button.usa-button-outline');
+    tree.find('button.usa-button-outline').simulate('click');
     expect(saveSpy.called).to.be.false;
   });
 
   it('should not call saveAddress when Update is clicked with invalid data', () => {
     saveSpy.reset();
-    const component = ReactTestUtils.renderIntoDocument(<AddressSection {...defaultProps}/>);
-    const tree = getFormDOM(component);
+    const tree = mount(<AddressSection {...defaultProps}/>);
 
     // Start editing
-    tree.click('button.usa-button-outline');
+    tree.find('button.usa-button-outline').simulate('click');
 
     // Clear out country to get a validation error
-    tree.fillData('[name="country"]', '');
+    tree.find('select[name="country"]').simulate('change', { target: { value: '' } });
```

Comment:
Seems like our helpers are still sort of useful, even with enzyme.

Reply:
Agreed. I was considering mirroring or refactoring the helpers so we can use them with Enzyme, but I wasn't going to do that in this PR.